### ISSUE TITLE
Ability to parse offline config file

### ISSUE THREAD

**Issue by maaaaz** [nesanton/fortiget#1]
Hello there,

Whoa, I haven't tested your scripts but they seem to do a pretty good job: adding the ability to parse offline config file would be awesome.
I saw you forked one of my project, feel free to reuse some code for yours :)

You also might want to check these ones:
- https://github.com/conix-security/springbok
- https://github.com/Fatal-Halt/FortiGate-Config-Parser
- http://jmanteau.fr/informatique/python/parsing-fortigate-configuration-in-python/

Cheers.

**Comment by nesanton** [comment]
Hi maaaaz,

Indeed a good thing to implement, although it'd need a great deal of changes. Well, looking at how other people do things (thanks very much for the links!), these scripts are in need of a good rewrite anyway. Maybe I should just query a full conf at the start, so then it does not matter much if it's a file or ssh session.

Thanks!

**Comment by maaaaz** [comment]
Yes, just add two options, one taking a local file, the second taking an IP+port+creds.
